http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -169,6 +169,15 @@ while sudo fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1
     sleep 5
   done
 
+# Install updated R version
+echo "Installing R 4.4.0..."
+# Add the CRAN repository to the sources list
+echo "deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" | sudo tee /etc/apt/sources.list -a
+# Update package list
+sudo apt-get update
+# Install new R version
+sudo apt install --no-install-recommends -y r-base=4.4.0-1.2004.0
```

Comment:
Nit, but could you store `4.4.0-1.2004.0` in an `R_Version` variable to make it easier to change, and also echo?
something like
R_version = `4.4.0-1.2004.0`

echo "Installing R ${R_version}"
...
sudo apt install --no-install-recommends -y r-base=${R_version}

Reply:
oh yeah that makes sense!